CONTRIBUTING.md

```diff
@@ -115,14 +115,26 @@ This is a list of accepted request-for-comments (RFC) documents by date of creat
 - [Error Mitigation by Subspace Expansion](https://docs.google.com/document/d/1JyQAwiw8BRT_oucZ6tQv0id6UhSdd3df1mNSPpOvu1I) by Ammar Jahin, Dariel Mok , Preksha Naik, Abdulrahman Sahmoud (@bubakazouba) Apr 28, 2023
 - [Implementation RFC for Mitiq calibration](https://docs.google.com/document/d/1EZUJyEEUQUH33UOgSIzCCvXyxP0WLOQn11W0x4Ox4nY/edit) by Andrea Mari (@andreamari) Nov 2, 2022
 - [Calibration tools for error mitigation RFC (abstract general solutions)](https://docs.google.com/document/d/1otUHnTlyNS-0rxGAxltHLF1iD5C9qT9oEZ3jn8VHWgw/edit) by Andrea Mari (@andreamari) Oct 6, 2022
-- [Identity insersion scaling RFC](https://docs.google.com/document/d/1hbd9frjYiSy0WujA0iCccc-oMO4Q-kZc2G4b3lkJHdk/edit) by Purva Thakre (@purva-thakre) Jun 29, 2022
+- [Identity insertion scaling RFC](https://docs.google.com/document/d/1hbd9frjYiSy0WujA0iCccc-oMO4Q-kZc2G4b3lkJHdk/edit) by Purva Thakre (@purva-thakre) Jun 29, 2022
 - [Readout Confusion Inversion RFC](https://docs.google.com/document/d/1buO5PrO5sS02VXjcaYf37RuR0rF6xpyr4J9H1tI4vN4/edit) by Amir Ebrahimi (@amirebrahimi) Jun 16, 2022
 - [Documentation reorganization RFC](https://docs.google.com/document/d/13un5TZPknSOhmOBkrL2rsofjGfdp2jDnd-DywLpGFPc/edit) by Ryan LaRose (@rmlarose) Dec 1, 2021
 - [Learning-based PEC RFC](https://docs.google.com/document/d/1VItesy6R5SlUa_YXW1km7IjFZ8kzyFeHUepHak1fEh4/edit) by Misty Wahl (@Misty-W) Oct 25, 2021
 - [Digital dynamical decoupling RFC](https://docs.google.com/document/d/1cRwFCTn6kUjI1P0kNydtevxIYtE4r8Omd_iWK0Pe8qo/edit) by Aaron Robertson (@Aaron-Robertson) Jan 28, 2021
 
+### Checklist for adding an approved QEM Technique
+
+After your RFC is accepted, the proposed feature (for example, a new QEM Method) will require the following:
+
+- Add the new QEM method to `mitiq/abbreviated_name_of_qem_method` such that the corresponding units tests are in `mitiq/abbreviated_name_of_qem_method/tests`
+- The code must follow the formatting and style guidelines discussed [ppreviously](#style-guidelines),
+- The new module has to be added to the [](apidoc.md). For more information, see [](contributing_docs.md#automatically-add-information-from-the-api-docs),
+- Add documentation for the new QEM method, additional details are available in [](contributing_docs.md#adding-files-to-the-user-guide),
+- In addition, update `docs/source/guide/glossary.md` with a one-line summary of what your new feature accomplishes, and
+- Update [](./readme.md#quick-tour) section of `README.md` with information related to your new feature.
+
+
 ## Code of conduct
-Mitiq development abides to the [Contributors' Covenant](https://mitiq.readthedocs.io/en/latest/code_of_conduct.html).
+Mitiq development abides to the [](./code_of_conduct.md#contributor-covenant-code-of-conduct).
```

Comment:
Unnecessary since the anchor is the title.
```suggestion
Mitiq development abides to the [](./code_of_conduct.md).
```

Reply:
This was not working when I was checking the changes locally. The link was dead. 

I had to make sure the main heading was specified after `.md`.

Reply:
What error are you seeing? I just tried locally and it's working!

Reply:
No error. The line appears as non-clickable as if we sandwiched the path of the md file in backticks. 

If it works for you, I could give it a try and check it in the RTD build.

Reply:
Looks good to me (same as what I see locally): https://mitiq--2382.org.readthedocs.build/en/2382/contributing.html#code-of-conduct